tools/scoring/README.md

```diff
@@ -10,4 +10,35 @@ To install the dependencies, please run the `python3 setup.py install` from the
 
 ## Usage
 
-From the `scoring` directory: `python3 scorer.py -ont path/to/ontology/yaml/resources -prop path/to/proposed/file.yaml -sol path/to/solution/file.yaml -v True`
+The application can be run from the command line. It takes three arguments:
+1. `-ont/--ontology` (required): Absolute path for the directory which contains your ontology
+2. `-prop/--proposed` (required): Absolute path for your proposed configuration file (to be scored)
+3. `-sol/--solution` (required): Absolute path for your solution configuration file
+
+Example: `scoring % python3 scorer.py -ont path/to/ontology/yaml/resources -prop path/to/proposed/file.yaml -sol path/to/solution/file.yaml`
```

Comment:
what is the percentage?

Reply:
Just the character my terminal uses. Should I change it to `$`?

Reply:
yes just start with `python3 scorer.py ...` ?